tutorials/3d/vertex_displacement_with_shaders.rst

```diff
@@ -190,11 +190,11 @@ we increase the frequency each level, decrease the amplitude, and calculate a ne
 ::
 
   float fbm(vec2 x) {
-    float height = 0.0;
-    float amplitude = 0.5;
-    float frequency = 3.0;
+    float h = 0.0;
+    float a = 0.5;
+    float p = 3.0;
```

Comment:
I feel like it'd be better if we used the longer variable names "height", "amplitude" and "frequency", for the sake of readability.

Reply:
Yes, I've updated that as well. Please look at the new commit.

Reply:
👍